### PR TITLE
intel-tensorflow is built with MKL support

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

File: thoth/adviser/wraps/mkl_threads.py [hunk 1]
```diff
@@ -57,5 +57,9 @@ def should_include(cls, builder_context: "PipelineBuilderContext"):
 
     def run(self, state: State) -> None:
         """Check for libraries using PyTorch."""
-        if "torch" in state.resolved_dependencies or "pytorch" in state.resolved_dependencies:
+        if (
+            "torch" in state.resolved_dependencies
+            or "pytorch" in state.resolved_dependencies
+            or "intel-tensorlfow" in state.resolved_dependencies
```
Comment (pacospace) [inline thoth/adviser/wraps/mkl_threads.py:63]: `intel-tensorlfow` typo :)

Comment (fridex) [reply thoth/adviser/wraps/mkl_threads.py]: nice catch, #fatfingers :raised_hand_with_fingers_splayed:

Comment (pacospace) [reply thoth/adviser/wraps/mkl_threads.py]: https://media.giphy.com/media/xT5LMPPfGcFXIGE1IQ/giphy.gif

**Comment by goern** [comment]
recheck

**Pull request merged**